packages/web-components/fast-element/src/templating/binding.ts

```diff
@@ -217,23 +217,23 @@ function updateTokenList(
                 continue;
             }
 
-            versions[currentName] = currentVersion;
+            classVersions[currentName] = version;
             tokenList.add(currentName);
         }
     }
 
-    state.v = currentVersion + 1;
+    state.v = version + 1;
```

Comment:
This is the line where the actual fix was. It's hard to see because I renamed the variables for clarity.